cmd/bosun/expr/funcs.go

```diff
@@ -244,6 +250,22 @@ func Duration(e *State, T miniprofiler.Timer, d string) (*Results, error) {
 	}, nil
 }
 
+func DropLe(e *State, T miniprofiler.Timer, series *Results, threshold float64) (*Results, error) {
+	for _, res := range series.Results {
+		nv := make(Series)
+		for k, v := range res.Value.Value().(Series) {
+			if float64(v) > threshold {
+				nv[k] = v
+			}
+		}
+		if len(nv) == 0 {
+			return nil, fmt.Errorf("drople: series %s is empty", res.Group)
```

Comment:
actually,
if i want to explicitly filter out all values that i don't want, then I expect to get an empty series back if none of the points match my criteria.  I left this as error because dropna behaves like this, but i would just like to get an empty series back and do my logic in my alerting rule on it.